src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -328,9 +333,18 @@ private function renderRowSeparator()
             return;
         }
 
-        $markup = $this->style->getCrossingChar();
+        if (self::SEPARATOR_MID === $type) {
+            list($leftChar, $midChar, $rightChar) = array($this->style->getCrossingMidLeftChar(), $this->style->getCrossingChar(), $this->style->getCrossingMidRightChar());
```

Comment:
not sure if we allow short array syntax for destructuring.. AFAIK it could be `[$a, $b, $] = array(...)`.

Reply:
Not sure if destructuring makes sense here, creating an array + destructuring seems like decreasing code readability compared to using 3 assignments.

Reply:
After a quick search, array destructuring doesn't look to be used in the code base.